plugins/rbac-backend/src/plugin.ts

```diff
@@ -53,7 +53,12 @@ export const rbacPlugin = createBackendPlugin({
         tokenManager,
       }) {
         const winstonLogger = loggerToWinstonLogger(logger);
-
+        const isRbacPluginEnabled =
+          config.getOptionalBoolean(`permission.enabled`);
+        if (!isRbacPluginEnabled) {
+          winstonLogger.info(`rbac backend plugin is disabled`);
+          return;
+        }
```

Comment:
I like the idea of this change. However, I don't believe it is working like we think it should as the RBAC plugin is still building.

If I set `permission.enabled: false` and check the database, I can see that the permissions and roles were added to the database.

Reply:
I am not really finding anywhere that would be a good place for this change to go that will prevent the RBAC plugin from building. We might have to look upstream to see how the permission framework is enabled / disabled.

Reply:
Yes, you're right, I should revert this change.

Reply:
We need separated pr for this purpose.